yt_dlp/extractor/animeonegai.py

```diff
@@ -0,0 +1,103 @@
+import uuid
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    determine_ext,
+    int_or_none,
+    str_or_none,
+    traverse_obj,
+    url_or_none,
+)
+
+
+class AnimeOnegaiIE(InfoExtractor):
+    _NETRC_MACHINE = 'animeonegai'
+    _VALID_URL = r'https?://(?:www\.)?animeonegai\.com/(?P<language>[a-z]{2})/watch/(?P<id>\w+)'
+    _GEO_COUNTRIES = ['MX']
+    _TESTS = [{
+        'url': 'https://www.animeonegai.com/es/watch/4Bpd6m6gGYVxdspKn?time=0&serie=true',
+        'md5': 'TODO: md5 sum of the first 10241 bytes of the video file (use --test)',
+        'info_dict': {
+            'id': '4Bpd6m6gGYVxdspKn',
+            'ext': 'mp4',
+            'title': 'Ep. 1: Eikichi Onizuka, 22 años: ¡Comienza la legendaria práctica docente del ex delincuente!',
+            'thumbnail': 'https://img-cdn.b-cdn.net/images/ayB0woggovEvoieXJ_1733001177.jpg',
+            'description': 'Eikichi Onizuka, de 22 años, fue un reconocido miembro de una pandilla de motociclistas durante su época de preparatoria. Ahora, su sueño de convertirse en profesor se hace realidad al comenzar su práctica docente. Espera con entusiasmo vivir un romance prohibido con una estudiante, pero le asignan una clase de jóvenes problemáticos. Allí conoce a la hermosa Nanako Mizuki, quien lo invita a su habitación después de clases. Sin embargo, pronto descubre que todo parece ser una trampa...',
+            'tags': ['Romance', 'Comedia', 'Drama', 'Serie'],
+            'series': 'GTO',
+            'episode_number': 1,
+        },
+        'skip': 'Login Required',
+    }]
+    _DEVICE_ID = str(uuid.uuid4())
```

Comment:
And then we can add a login hint
```suggestion
    _LOGIN_HINT = ('Use  --username token --password ACCESS_TOKEN  where ACCESS_TOKEN '
                   'is the "ott_token " from your browser local storage')
    _DEVICE_ID = str(uuid.uuid4())
```